tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -86,3 +86,34 @@ Wait Until All Builds Are Complete
     FOR    ${build_data}    IN    @{builds_data}
         Wait Until Build Status Is    namespace=${namespace}    build_name=${build_data['metadata']['name']}  expected_status=Complete
     END
+
+Delete Multiple Builds
+    [Documentation]     Deletes Multiple Builds in the given namespace
+    [Arguments]   @{build_name_list}  ${namespace}
+    FOR    ${build}    IN    @{build_name_list}
+        ${build_name}=  Search Last Build  namespace=${namespace}    build_name_includes=${build}
+        Delete Build    namespace=${namespace}    build_name=${build_name}
+    END
+
+Start Remaining Builds And Wait Until All Builds Are Complete
```

Comment:
Could be renamed to  `Rebuild Missing Or Failed Builds`

Reply:
Updated